smarts/core/scenario.py

```diff
@@ -116,7 +116,7 @@ class Via:
     required_speed: float
 
 
-@dataclass(frozen=True)
```

Comment:
Had to set the `mission.start_time` to 0 otherwise smarts.reset will still step through time_steps until original mission.start_time